docs/source/developer_guides/lora.md

```diff
@@ -272,6 +272,50 @@ trainer = Trainer(
 )
 ```
 
+## Efficiently train tokens alongside LoRA
+
+Sometimes it is necessary to not only change some layer's weights but to add new tokens as well. With larger models this can be a memory-costly endeavour. PEFT LoRA adapters support the `trainable_token_indices` parameter which allows tuning of other tokens alongside fine-tuning of specific layers with LoRA. This method only trains the tokens you specify and leaves all other tokens untouched. This saves memory and doesn't throw away learned context of existing token embeddings in contrast to when training the whole embedding matrix. Under the hood this method uses the [`~TrainableTokenLayer`].
```

Comment:
This link is also broken :/

![image](https://github.com/user-attachments/assets/4ea9ba2f-6e0c-4d9b-a397-a77fad297a71)